src/clang-cpp-frontend/clang_cpp_convert.cpp

```diff
@@ -622,3 +877,67 @@ bool clang_cpp_convertert::get_template_decl(
 
   return false;
 }
+
+bool clang_cpp_convertert::search_this_map(
+  const std::size_t address,
+  this_mapt::iterator &this_it)
```

Comment:
Is this separate function really necessary? The expression `this_map.find(address) != this_map.end()` is equivalent to it.

Reply:
Done. Removed